src/deep-blue-green/backend/src/replication/start/bootstrap.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * Created by CCristi on 2/10/17.
+ */
+
+/* eslint no-use-before-define:0 */
+
+'use strict';
+
+const AWS = require('aws-sdk');
+const CHUNK_SIZE = 100;
+
+exports.handler = (event, context, callback) => {
+  const tableManager = new TableManager(new AWS.DynamoDB(), event.Table);
+  const backFillExecutor = new BackFillExecutor(tableManager, event.Hash);
+  const scanPayload = {};
+
+  scanPayload[':hash'] = {
+    S: event.Hash,
```

Comment:
Please use the following node module for all build/read ops in/from DDB: https://github.com/CascadeEnergy/dynamoDb-marshaler